README.md

```diff
@@ -1,44 +1,49 @@
+### 2. [Researchers](research.md)
+From this stage on, progress is tracked exclusively through the [Data Rescue archivers app](http://archivers.space). Researchers inspect individual URL's/datasets and make a more granular assessment of them, including a preliminary assessment of the type of data, the kinds of tools that might be useful for harvesters, and the importance of the data.  [Research.md](research.md) describes this process in more detail. 
+*Often this step is incorporated into either "Seeding and Sorting" or "Harvesting".*
 
-### 3. [Harvesters](harvesting-toolkit)
-Harvesters take the "uncrawlable" data and try to figure out how to actully capture it based on the recommendations of the Researchers.  This is a complex task which can require substantial technical expertise, and which requires different techniques for different tasks. Harvesters should see the included [Harvesting Toolkit](./harvesting-toolkit) for more details and tools. 
+### 3. [Harvesters](https://github.com/edgi-govdata-archiving/harvesting-tools/)
+Harvesters caputre the actual data. This is a complex task which can require substantial technical expertise, and which requires different techniques for different tasks. The [harvesting tools](https://github.com/edgi-govdata-archiving/harvesting-tools) repo has a number of starting points and describes the required harvesting format. **TODO: I don't think this is currently true, but I want to reduce the duplication of documentation!!! Maintenance is becoming a serious issue. ** 
 
 ### 4. [Checkers](checking.md)
 Checkers inspect a harvested dataset and make sure that it is complete. The main question the checkers need to answer is "will the bag make sense to a scientist"? Checkers need to have an in-depth understanding of harvesting goals and potential content variations for datasets.
 
 ### 5. [Baggers](bagging.md)
-Baggers do some quality assurance on the dataset to make sure the content is correct and corresponds to what was described in the spreadsheet. Then they package the data into a bagit file (or "bag"), which includes basic technical metadata and upload it to final DataRefuge destination.
-
+- Do some quality assurance on the dataset to make sure the content is correct and corresponds to what was described in the spreadsheet
+- Package the data into a bagit file (or "bag"), which includes basic technical metadata and upload it to final DataRefuge destination.
 
 ### 7. [Describers](metadata.md)
-Describers creates a descriptive record in the DataRefuge CKAN repository for each bag. Then they links the record to the bag, and make the record public
-**********************
+Describers creates a descriptive record in the destination CKAN repository for each bag. Then they links the record to the bag, and make the record public.
+
+## After the Event!
```

Comment:
Do we want this here or in a checklist? I think we could remove the checklist we have form the overview and add it here, or have like all checklist/runsheet items out of this repo (including advance work) and somewhere else. I know the EDI event toolkit has a bunch and maybe that is where it should live